### PR TITLE
Polymer 3

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
ready for eyes

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1 +1,3 @@
 bower_components-1.x
+test/acceptance/*
```
[the remark this answers is not in the document]
Comment (awikkerink) [reply .eslintignore]: test/acceptance shouldn't be in .gitignore. This entry is just for Galen tests, if they exist

File: d2l-user-card-auto.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/*
+`d2l-user-card-auto` is a Polymer-based web component for creating a user card which automatically fills itself out when given a token and url.
+*/
+/*
+  FIXME(polymer-modulizer): the above comments were extracted
```
[the remark this answers is not in the document]
Comment (awikkerink) [reply d2l-user-card-auto.js]: The convention is to leave the FIXME until everything is moved from hybrid to polymer 3, to make the reconversion/applying of patches easier if necessary.

File: polymer.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
+  "npm": true,
   "lint": {
-    "rules": ["polymer-2-hybrid"]
+    "rules": ["polymer-3"]
   },
   "sources": [
     "d2l-user-card.html",
```
[the remark this answers is not in the document]
Comment (awikkerink) [reply polymer.json]: Hmm. I guess this is not part of the `toPolymer3.sh` script.

**Comment by ctwomey1** [comment]
Demo seems to work, Changes look reasonable to me.

**Comment by jstefaniuk-d2l** [comment]
yarg squash merge conflicts

**Pull request merged**